### DISCUSSION TITLE
"Bulk unwatch" repos feature, or better control of auto-subscriptions

### DISCUSSION THREAD

**Discussion by anentropic** [community/community#34724] [Repositories] [answered]
There is an annoying behaviour currently which affects members of organizations with lots of repositories

When you get added to a group it seems that you get auto-subscribed to all the repos you have access to

This results in me receiving an email like:

> [GitHub] Subscribed to 100 MyOrg repositories
> 
> Hey there, we’re just writing to let you know that you’ve automatically started watching several repositories on GitHub.
>
> You’ll receive notifications for all issues, pull requests, and comments that happen inside the repository. If you would like to stop watching any of these repositories, you can manage your settings here:
>
> [... individual links to subscription settings for 100 repos ...]

And then a deluge of notifications for every activity on each of the repos.

Typically there are only a handful of repos that are directly relevant to my current work (for which I would like to receive notifications), but it's quite reasonable that I get added to a "Engineers" group that grants access to all of them.

The annoying part is two-fold:

- no way to bulk unwatch all these repos I just got added to
- it would be great if there was a preference to opt out of getting auto subscribed to new repos in this way

Now I have to individually go through 90+ repos and unwatch them

**Comment by anentropic** [comment]
hmm ok, I just found this:
![Screenshot 2022-09-30 at 16 12 06](https://user-images.githubusercontent.com/147840/193301430-3c298465-3731-44f3-a031-65944b28167e.png)

so all I need is the bulk unwatch feature

**Comment by jfullstackdev** [comment, marked as answer]
yes, you can set the default behavior at your settings, 

https://github.com/settings/notifications

plus this for the bulk

https://github.com/watching

you must be logged-in both

**Comment by anentropic** [comment]
Thank you!

It would be really helpful to have these two links in the "You've just been subscribed to 100 repos" email
